logic/CMakeLists.txt

```diff
@@ -52,6 +61,20 @@ SET(LOGIC_SOURCES
 	wonko/OpSys.h
 	wonko/OpSys.cpp
 
+	# Resources
```

Comment:
Again, I could go ahead and use this for other things, but that would mean a bigger diff...